functions/src/index.ts

```diff
@@ -12,13 +12,70 @@ const db = admin.database(app)
 export const helloWorld = functions.https.onRequest(async (request, response) => {
   const usersRef = db.ref("users")
   await usersRef.update({
-    alanisawesome: {
-      nickname: "Alan The Machine",
+    kilvishId1: {
+      kilvishId: "kilvishId1",
+      name: "Radhey",
+      email: "radheyit@gmail.com",
+      phone: "8983026208",
     },
-    gracehop: {
-      nickname: "Amazing Grace",
+    kilvishId2: {
+      kilvishId: "KilvishId2",
+      name: "dummy Name",
+      email: "adminji@yopmail.com",
+      phone: "8208567820",
     },
   })
-  const snapshot = await db.ref("users/alanisawesome").get()
+  console.log(request.params);
+  const snapshot = await db.ref("users/kilvishId1").get()
   response.send(snapshot.val())
 })
+
+export const verifyOtp = functions.https.onRequest(async (request, response) => {
+  const reqBody = JSON.stringify(request.body);
+  const data = JSON.parse(reqBody)['data'];
+
+  const kilvishId = data["kilvishId"];
+  const phoneOtp = data["phoneOtp"];
+  const emailOtp = data["emailOtp"];
+
+  const snapshot = await db.ref(`users/${kilvishId}`).get()
+  if (snapshot != null && snapshot.val()) {
+    if (snapshot.val()["verifyPhone"] == false && snapshot.val()["verifyEmail"] == false) {
+      if (phoneOtp == "1234" && emailOtp == "5678") {
+        response.send({ "data": { "success": true } })
```

Comment:
I think this is next step. We need to send token generated from admin sdk for the kilvish id & then set that in the frontend.